version.json

```diff
@@ -10,7 +10,8 @@
   },
   "publicReleaseRefSpec": [
     "^refs/heads/main$",
-    "^refs/heads/v\\d+(?:\\.\\d+)?$"
+    "^refs/heads/v\\d+(?:\\.\\d+)?$",
```

Comment:
The latest one wasn't here because we're not building on tags. Either way we're always forcing this build option on builds in pipeline, so no worries about this.